step.yml

```diff
@@ -0,0 +1,95 @@
+---
+title: Xcode test shard calculation
+summary: Calculates the test shards for your Xcode tests.
+description: |-
+  TBD
+website: https://github.com/bitrise-steplib/bitrise-step-xcode-test-shard-calculation
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-xcode-test-shard-calculation
+support_url: https://github.com/bitrise-steplib/bitrise-step-xcode-test-shard-calculation
+
+type_tags:
+- test
+
+project_type_tags:
+- ios
+- macos
+- react-native
+- cordova
+- ionic
+- flutter
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-xcode-test-shard-calculation
+
+inputs:
+- product_path:
+  opts:
+    title: Product path
+    summary: |-
+      The path of the test bundle.
+    description: |-
+      The path of the test bundle.
+      
+      The step supports the following formats:
+      - xcresrun
+      - xctestproducts
+      
+      It will use the specified file to collect the built tests and generate the test shards.
+    is_required: true
+
+- shard_count:
+  opts:
+    title: Shard count
+    summary: |-
+      The number of test shards to create.
+    description: |-
+      The number of test shards to create.
+      
+      The output folder will contain `shard_count` number of files, each containing the tests to run in that shard.
+    is_required: true
+
+- shard_calculation: "alphabetically"
```

Comment:
At the moment this is the only strategy so this input would not be needed right now technically. But maybe it is better to already highlight that we will be supporting more.